source/frontend/properties/gc/Material/StageSurface.cpp

```diff
@@ -1,7 +1,7 @@
 #include "Common.hpp"
 #include <frontend/properties/gc/TevSolver.hpp> // optimizeNode
 #include <frontend/widgets/Image.hpp>           // ImagePreview
-#include <librii/hx/KonstSel.hpp>               // elevateKonstSel
+#include <librii/hx/KonstSel.hpp> // elevateKonstSel
```

Comment:
Please don't break the formatting of unrelated code. The files here are run through clang-format to ensure a consistent style across the whole project.